airmozilla/starred/tests/test_views.py

```diff
@@ -227,3 +227,53 @@ def test_anonymous_starred_events(self):
         ok_(event1.title in response.content)
         ok_(event2.title in response.content)
         ok_(event3.title not in response.content)
+
+    def test_event_pagination(self):
+        url = self.home_url
+        user = self._login(username='lisa')
+
+        events = []
+
+        for i in range(25):
+            event = self.create_event("Event Title %d" % (i + 1))
+            StarredEvent.objects.create(user=user, event=event)
+            events.append(event.id)
+
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        for id in range(len(events)):
+            match = 'data-id="%d"' % events[id]
+            if id < 10:
+                ok_(match in response.content, id)
+            else:
+                ok_(match not in response.content, id)
+
+        ok_('href="%s"' % reverse('starred:home', args=(2,)) in response.content)
+        ok_('href="%s"' % reverse('starred:home', args=(0,)) not in response.content)
```

Comment:
To avoid making these lines too long, you can perhaps just set, before these two lines, something like:
```python
output = response.content
```
That's 10 characters less. Then the lines don't get too long.